Makefile

```diff
@@ -696,6 +703,38 @@ generate_yaml_new: $(HELM) $(HOME)/.helm helm-repo-add
 	./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
 	(cd install/kubernetes/helm/istio; ${ISTIO_OUT}/istioctl gen-deploy -o yaml --values values.yaml)
 
+generate_e2e_test_yaml: $(HELM) $(HOME)/.helm helm-repo-add
+	$(HELM) dep update --skip-refresh install/kubernetes/helm/istio
+	./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
+	cat install/kubernetes/namespace.yaml > install/kubernetes/istio.yaml
+	$(HELM) template --set global.tag=${TAG} \
+		--name=istio \
+		--namespace=istio-system \
+		--set global.hub=${HUB} \
+		--set global.proxy.enableCoreDump=${ENABLE_COREDUMP} \
+		--set global.proxy.concurrency=1 \
+		--set prometheus.scrapeInterval=1s \
+		--set gateways.istio-ingressgateway.autoscaleMax=1 \
+		--set mixer.policy.replicaCount=2 \
+		--set mixer.policy.autoscaleEnabled=false \
+		--values install/kubernetes/helm/istio/values.yaml \
+		install/kubernetes/helm/istio >> install/kubernetes/istio.yaml
+
+	cat install/kubernetes/namespace.yaml > install/kubernetes/istio-auth.yaml
+	$(HELM) template --set global.tag=${TAG} \
+		--name=istio \
+		--namespace=istio-system \
+		--set global.hub=${HUB} \
+		--set global.mtls.enabled=true \
+		--set prometheus.scrapeInterval=1s \
+		--set gateways.istio-ingressgateway.autoscaleMax=1 \
+		--set mixer.policy.replicaCount=2 \
```

Comment:
i notice in the e2e_mixer results in circleci that only one policy instance ever runs. Does this helm option not exist (or has it been changed) in `master`?

Reply:
Yeah it looks like circle is using something else.
https://github.com/istio/istio/blame/release-1.1/.circleci/config.yml#L248